subprojects/docs/src/docs/userguide/migration/migrating_from_groovy_to_kotlin_dsl.adoc

```diff
@@ -131,6 +131,36 @@ dependencies {
 
 While staying valid Groovy, it is now unambiguous and close to the Kotlin syntax, making it easier to then rename the script to turn it into a Gradle Kotlin DSL script.
 
+It is important to note that while Groovy extra properties can be modified using an object's `ext` property, in Kotlin they are modified using the `extra` property. It is important to look at each object and update the build scripts accordingly.
+
+For example,
```

Comment:
Instead of adding inline untested snippets here I think it would be better to link to the reference documentation at https://docs.gradle.org/current/userguide/kotlin_dsl.html#extra_properties

It shows how to define an extra property but also how to reference one.

Reply:
Hi @eskatos , sorry for late response, I was travelling.
You mean to add the link in the "for example" instead of the example. And maintain like 134 as it is. Is this correct?

Reply:
Yes exactly!